C-Plus-Plus/search/TernarySearch.cpp

```diff
@@ -0,0 +1,86 @@
+#include <bits/stdc++.h> 
+using namespace std; 
+
+//Recursive Approach: Time Complexity Is O(log3(N))
+int ternarySearchRecursion(int l, int r, int key, int ar[]) 
```

Comment:
Use meaningful variables name instead of ```l```, ```r```. A meaningful variable name would help beginners to get grasp of the algorithm in no time.